ReceiveMessage/models/UserState.ts

```diff
@@ -1,11 +1,22 @@
-import mongoose, { Schema } from 'mongoose';
+import mongoose, { Schema, Document } from 'mongoose';
+import ChatbotMessage;
+
+export interface State extends Document {
+    userId: string;
+    currMessage: ChatbotMessage;
```

Comment:
Change this to be of type IMessage[_id]. What's stored here isn't actually an entire ChatbotMessage, but of the id of a chatbot message, like this in the article 
![image](https://user-images.githubusercontent.com/57023314/108137828-aa4e7b80-708a-11eb-9072-0a554739b8ba.png)